### PR TITLE
[15.0][MIG]-purchase_order_analytic_search

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
@ForgeFlow

### CODE REVIEW

**Review by AaronHForgeFlow — COMMENTED** [review]
The filter works, but it is only available in the request for quotation menu. Can you add it in the purchase order menu as well?

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

**Review by JoanSForgeFlow — APPROVED** [review]
LGTM!

**Review by hieulucky111 — APPROVED** [review]
LGTM
Code review

File: purchase_order_analytic_search/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @api.depends("order_line.account_analytic_id")
+    def _compute_analytic_accounts(self):
+        for purchase in self:
+            purchase.account_analytic_ids = purchase.mapped(
+                "order_line.account_analytic_id"
+            )
+
+    @api.model
+    def _search_analytic_accounts(self, operator, value):
+        po_line_obj = self.env["purchase.order.line"].filtered(
```
Comment (AaronHForgeFlow) [inline purchase_order_analytic_search/models/purchase_order.py:21]: Maybe it is best to define a base_domain and then use the expression.add to extend the domain?

**Pull request merged**